docs/reference/modules/gateway.asciidoc

```diff
@@ -56,9 +56,22 @@ NOTE: These settings only take effect on a full cluster restart.
 [[dangling-indices]]
 ==== Dangling indices
 
-When a node joins the cluster, any shards stored in its local data
-directory which do not already exist in the cluster will be imported into the
-cluster. This functionality is intended as a best effort to help users who
-lose all master nodes. If a new master node is started which is unaware of
-the other indices in the cluster, adding the old nodes will cause the old
-indices to be imported, instead of being deleted.
+When a node joins the cluster, if it finds any shards stored in its local data
+directory that do not already exist in the cluster, it will consider those
+shards to be "dangling". You can automatically import dangling indices
+into the cluster using the following setting, however this is not safe, and
+you should use the <<dangling-indices-api,Dangling indices API>> instead.
```

Comment:
```suggestion
shards to be "dangling". Importing dangling indices
into the cluster using `gateway.auto_import_dangling_indices` is not safe. 
Instead, use the <<dangling-indices-api,Dangling indices API>>.
```

Reply:
I think that this revision is more strongly worded and indicates that users should definitely avoid this setting in favor of the API.